### PR TITLE
Update multiple files to match 2018 curriculum

### PR DESCRIPTION
**Pull request by catheraaine**
- Rework project outline
- Add BEM resource
- Add project file with homework descriptions
- Add Career Prep section to Phase 2
- Refactor linkage

### CODE REVIEW

File: projects/proj-event-cards.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+We want to build these cards with the following in mind:
+- A "Card" is a small piece of a website, not usually the whole page. We'll probably want to put multiple cards on the final page.
+- Put things in the order that makes the most sense for HTML.
+- Research semantic tags for each item; it is unlikely you'll be using `<div>` very often, but you may decide that it's the best choice some places.
+- Creating [accessible elements may require additional attributes](https://www.w3.org/WAI/standards-guidelines/wcag/).
+- Inline comments are helpful for reminding yourself why you made a decision you made.
+- Focus on semantically strong, accessible Markup. Styles will come later, and if you do them now, we'll probably completely ignore them anyway.
+
+Next week we will spend 5 minutes or so going over the HTML choices each of you have made. Try to do this individually (not as a group). Please do take advantage of office hours or pairing with developers!
```
Comment (cromwellryan) [inline projects/proj-event-cards.md]: `office hours` is a thing we might want to link to in the sparkbox/standard.

File: projects/proj-event-cards.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+```html
+[[[https://<url to your event card>]]]
+```
+
+You will also need to link to your pen as an external css resource by going to *Settings -> CSS - > Add External Stylesheets/Pens* and adding the URL of your card.
+
+### Browser Testing
+*Note*: This is a big ask. It's not fun. It won't be perfect. You don't have the full resources of a design team. Do the best you can, but don't burn out.
+
+At Sparkbox, we don't have Quality Assurance, so we usually do our own browser testing. If there is interest, we can do a 'bug bash' together, to learn about the device wall and [BrowserStack](https://www.browserstack.com/start).
```
Comment (cromwellryan) [inline projects/proj-event-cards.md]: Link QA... possible destination https://en.wikipedia.org/wiki/Quality_assurance#Software_development

File: projects/proj-event-cards.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+
+#### Gold Level Support
```
Comment (cromwellryan) [inline projects/proj-event-cards.md]: Not a hill I'll die on, but I can see this list changing at a different cadence than the rest of the curriculum. For that reason, you might want to put this in a different file with a link.

File: projects/proj-event-cards.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+Your homework this week is comprised of a few smaller tasks.
+
+#### BEM
+From this point forward, you'll be expected to use BEM for any CSS and HTML you write. Fork your Grids and refactor all of your classes to use BEM. Then, pair with one of our BEM devs to review your *finished* BEM.
+
+#### SVG or `srcset`
+Spend some time playing with images or SVGs. Fork and update your event card to use either an SVG icon (Next to your address or phone number is nice, or whatever you like) OR update your images to use `srcset`.
+
+#### Personal Websites
+It's about the time in your apprenticeship when you should be looking to find a permanent web development role (if you are interested in that sort of thing). In your career prep sessions, you've hopefully developed skills to help you improve your marketability. Spend some time reworking or redoing your personal websites, cleaning up your git profiles, etc. We're here to help, so pair!  Remember to take advantage of pairing time not only with developers, but also with our wonderful Project Management, Content, and Design teams.
```
Comment (cromwellryan) [inline projects/proj-event-cards.md]: @silverli, can we get a short blip about _career prep sessions_ somewhere in here that @catheraaine can link to?

File: projects/proj-event-cards.md (hunk 1 above)
Comment (catheraaine) [reply projects/proj-event-cards.md]: I  think we'll need to add this to the Standard, I'm not finding a section about office hours!

File: index.md [hunk 5]
```diff
@@ -47,56 +47,65 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
...
+  * Finish reading [Responsive Web Design][] by Ethan Marcotte
+* Week 3: Layout
+  * Finish [BEM by Example][] by Nathan Rambeck
+* Week 4: BEM + SVG
+  * [CodeSchool: Assembling Sass][]
+  * [CodeSchool: Assembling Sass Part 2][]
+* Week 5: Sass
+* *Extra Credit: [FizzBuzz][] in CSS on [CodePen][]*
+
+##### Career Preparedness
```
Comment (silverli) [inline index.md]: can you say Career Preparedness Workshop? (just so it's clear) :)

File: index.md [hunk 6]
```diff
@@ -47,56 +47,65 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
...
+
+Learning development skills is a crucial part of the apprenticeship curriculum, but we also want the apprentices to feel prepared and fully equipped to be applying for and entering the web industry at the end of their time at Sparkbox. This workshop will help apprentices learn how to highlight their strengths, and how to approach the job market of the web industry.
+
+Over 4 weeks, we cover the following using both lecture and hands on activities:
+  * Part 1: Introduction and Resumes
+  * Part 2: How to Really Read a Developer Job Posting and Cover Letters
+  * Part 3: Interview Preparation and Salary Negotiation
+  * Part 4: Open Job Preparedness work hour and Q&A Session
+    This is just an open hour to ask any lingering questions, and continue working on their resumes, cover letter, applying for jobs, and their web portfolio site.
```
Comment (silverli) [inline index.md]: can you have this here instead for the 4th point?:
> * Part 4: Open Job Preparedness work hour and Q&A Session. This is just an open hour to ask any lingering questions, and continue working on their resumes, cover letter, applying for jobs, and their web portfolio site.

**Pull request merged**